satellite/metainfo/metainfo_test.go

```diff
@@ -343,13 +343,62 @@ func TestCommitSegment(t *testing.T) {
 				ExpirationDate: expirationDate,
 			}
 
-			limits := make([]*pb.OrderLimit, len(addresedLimits))
-			for i, addresedLimit := range addresedLimits {
-				limits[i] = addresedLimit.Limit
+			limits := make([]*pb.OrderLimit, len(addressedLimits))
+			for i, addressedLimit := range addressedLimits {
+				limits[i] = addressedLimit.Limit
 			}
 			_, err = metainfo.CommitSegment(ctx, "bucket", "path", -1, pointer, limits)
 			require.Error(t, err)
-			require.Contains(t, err.Error(), "less than or equal to the repair threshold")
+			require.Contains(t, err.Error(), "is less than or equal to the repair threshold")
+		}
+
+		{
```

Comment:
I still sort of have the vague feeling that repair threshold (the previous test) shouldn't matter because it _should_ always less than the success threshold.  Not a deal breaker, but more tests = slower testing.

Reply:
I don't think these tests are particularly intense, and having the two separate cases allows us to have more coverage. The first test is to make sure that the error we get when we are less than the repair threshold is the "less than the repair threshold" error. The new test is to test that the error we get when we are greater than the repair threshold but less than the success threshold is the "less than the success threshold" error. These tests are both useful regardless of the issue regarding other tests where the two thresholds are equal.

Reply:
yeah I think I agree with Moby